### ISSUE TITLE
Deno: error: npm package '@webviewjs/webview-freebsd-x64' does not exist.

### ISSUE THREAD

**Issue by lassejlv** [webviewjs/webview#9]
When running `deno install npm:@webviewjs/webview`, im getting: 

```bash
Add npm:@webviewjs/webview@0.1.2
error: npm package '@webviewjs/webview-freebsd-x64' does not exist.
```

**Comment by lassejlv** [comment]
Same with bun

<img width="765" alt="Screenshot 2024-11-09 at 17 13 38" src="https://github.com/user-attachments/assets/94e70b8c-fb75-40e3-96cd-117011e8fdc6">

**Comment by twlite** [comment]
Only the following platforms are supported at the moment:

| Platform                | Supported |
| ----------------------- | --------- |
| x86_64-apple-darwin     | ✅        |
| x86_64-pc-windows-msvc  | ✅        |
| i686-pc-windows-msvc    | ✅        |
| aarch64-apple-darwin    | ✅        |
| aarch64-linux-android   | ✅        |
| armv7-linux-androideabi | ✅        |
| aarch64-pc-windows-msvc | ✅        |

Bun installed the package successfully, but deno seems to error out early.

**Comment by lassejlv** [comment]
I am on aarch64-apple-darwin so ig its deno

**Comment by twlite** [comment]
> I am on aarch64-apple-darwin so ig its deno

yes, it's more related to deno resolving the packages. Bun seems to be working fine

**Comment by bbak** [comment]
Same here, running deno 2.2.12 on Win11: `error: npm package '@webviewjs/webview-freebsd-x64' does not exist.`

**Comment by Mqxx** [comment]
> Same here, running deno 2.2.12 on Win11: `error: npm package '@webviewjs/webview-freebsd-x64' does not exist.`

Same here
